MetabolicAtlas/MetabolicAtlas
# feat: add more details pages for GotEnzymes

**Related issue(s) and PR(s)**  
This PR closes #1032.

Joint work with @e0.


**Type of change**  

 
**List of changes made**  

- Add details pages for `protein`, `organism`, and `domain`.
- Update landing page to accommodate for the new details pages (search and examples).
- Hide column of current page's entity type in the enzymes table.
- Turn table cells from columns/fields that have details pages into links.

**Screenshot of the fix**  

<img width="1661" alt="image" src="https://user-images.githubusercontent.com/423498/175036600-e09bb73e-b5e4-45ee-be24-c5036cf213e8.png">
<img width="1660" alt="image" src="https://user-images.githubusercontent.com/423498/175036725-0795c75a-2137-4227-99d3-b6060a05f230.png">


**Testing**  

- Relative urls that can be reused both for production and local testing
- Instructions on how to test
1. Test handling of new details pages from the landing page `/gotenzymes`. Please try some searches as well as clicking on the provided example links.
2. Verify that the page loads without problems for pages that do not have basic info or cross references, for example `/gotenzymes/organism/hsa`.
3. Verify that the column matching the current details page, for example the `Organism` column on `/gotenzymes/organism/hsa`, is not shown.
4. Verify that the links from the enzymes tables are working correctly. Please try all 6 types.

**Further comments**  


**Definition of Done checklist**  
- [ ] My code meets the Acceptance Criteria (no new tests added)
